yaml/roads.yaml

```diff
@@ -235,14 +236,26 @@ filters:
         - {tags->ski: designated}
     min_zoom: 'LEAST(mz_calculate_path_major_route(osm_id), 13)'
     table: osm
+    extra_columns: [bicycle, foot, horse, tracktype, "tags->incline", "tags->trail_visibility", "tags->sac_scale"]
```

Comment:
Yeah, those should have been in output. Was this corrected later?

(What is `extra_columns` for?)